api/src/neo4j/queries/search/helper.js

```diff
@@ -0,0 +1,17 @@
+const INTEGRATED_MODELS = require('data/integratedModels');
+
+const MODELS = INTEGRATED_MODELS.map(m => ({
+  label: m.short_name.replace('-GEM', 'Gem'),
+  name: m.short_name,
+}));
```

Comment:
Just looking at this helper, it gives the vibe that it would be useful across the entire `api`, so maybe there's a better place for it?

Reply:
That could be the case but I generally prefer to refactor things only they are needed. For example if we want to use `MODELS` in another file later but they might need to be formatted differently.